modules/cmr_module/CmrApi.cc

```diff
@@ -116,27 +116,27 @@ const nlohmann::json &CmrApi::get_children(const nlohmann::json &jobj) const
     BESDEBUG(MODULE, prolog << json.probe_json(jobj) << endl);
     bool result = jobj.is_null();
     if(result){
-        stringstream msg;
-        msg <<  "ERROR: Json document is NULL: " << endl << jobj.dump(2) << endl;
-        BESDEBUG(MODULE, prolog <<  msg.str() << endl);
-        throw CmrInternalError(msg.str(), __FILE__, __LINE__);
+        string msg;
+        msg + "ERROR: Json document is NULL. json: " + jobj.dump();
```

Comment:
In these error messages, you're using the '+' operator as an assignment operator. Does that work? It's hard to look up and I cannot find a reference for it. You could rewrite the above as:

msg += "ERROR: ..." + jobj.dump();

Or, as:

string msg{"ERROR: ..."};
msg += jobj.dump;

Reply:
WTF - I had all those changes already...